unfoldingWord/translationCore
# Feature bspidel 2640

#### This pull request addresses:

Mock drop down list for Gateway Language Selection

#### How to test this pull request:

1 Start app
2 Select a Project
3 On tool selection page
  Note "Launch" button is selectable
4 select "Gateway Language" from drop down list
  Note "Launch" button is not selectable
5 Hover over "Launch" button
  Note Hint: 'Please select the Gateway Lanugage from which you would like to check your translation.'
6 Select "Hindi" from drop down list
  Note  "Launch" button is not selectable
7 Hover  over "Launch" button
  Note Hint: 'Only English is currently supported.'


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/3748)
